### ISSUE TITLE
Is there a way to circumvent the Fails WCAG SC 1.4.4 error?

### ISSUE THREAD

**Issue by kobe-f8** [barvian/fluid-tailwind#11]
I get this error on my latest project where i'm working with a lot of large font size that have to be reduced to a much smaller font size on mobile. Like this case for example: `~text-sm/4xl`.

But that gives me this error:
```
/* not fluid from 0.875rem at 40rem to 2.25rem at 80rem: Fails WCAG SC 1.4.4 at i.e. 200rem */
```

I'm I doing something wrong or could this be a bug?

**Comment by barvian** [comment]
Hey, thanks for filing. The warning is there because that text wouldn't enlarge to 2x its size if someone tried to 5x zoom the page at a screen width of i.e. 100rem, which would create [the accessibility violation](https://www.w3.org/WAI/WCAG21/Understanding/resize-text.html).

That being said, it'd probably be good to be able to opt out of these warnings. I'll think about this and keep this open for now!

**Comment by kobe-f8** [comment]
Thank you for looking into it and for the quick response 🙌🏻

**Comment by peterpalkovic** [comment]
> That being said, it'd probably be good to be able to opt out of these warnings.

This would be useful in some scenarios.

**Comment by barvian** [comment]
@kobe-f8 @peterpalkovic I'm thinking of adding an option you could pass to the plugin that would simply warn about these issues on the console but still output the fluid code. Would that work for your use cases or were you hoping for a different approach?

**Comment by krstivoja** [comment]
@barvian any update on this? This would be really good for some design cases.

**Comment by barvian** [comment]
@krstivoja Working on this feature for v0.2.0!

**Comment by barvian** [comment]
This feature is now live in v0.2.0 (see #15)! Feel free to comment in the discussion or open another issue if you find a bug. Thanks for the feedback 🙂
